index.html

```diff
@@ -154,7 +154,7 @@ <h2>Tools</h2>
               <p>Tools to assist on your digital journey</p>
               <div class="links">
                 <a target="_self" href="./vpn.html"> VPN Toolkit </a>
-                <a target="_self" href="./spa.html"> Privacy Quiz </a>
+                <a target="_self" href="./spa.html"> Anonimity Quiz </a>
```

Comment:
There's a quick typo here: 

```suggestion
                <a target="_self" href="./spa.html"> Anonymity Quiz </a>
```